src/mission_ui.cpp

```diff
@@ -123,10 +124,21 @@ void game::list_missions()
             y += fold_and_print( w_missions, point( 31, y ), getmaxx( w_missions ) - 33, col,
                                  miss->name() + for_npc );
 
+            auto format_tokenized_description = []( const std::string description,
+            std::vector<std::pair<int, std::string>> rewards ) {
+                std::string formatted_description = description;
+                for( size_t i = 0; i < rewards.size(); i++ ) {
+                    std::regex token("<reward_count:" + rewards[i].second + ">");
```

Comment:
You don't need a regex for simple string replacement, we have `string_replace` for this.

Reply:
I was utilizing it at one point, but refactored to this. I'll update to use `string_replace` instead.